src/EppoClient.php

```diff
@@ -121,11 +124,30 @@ public static function getInstance(): EppoClient
         return self::$instance;
     }
 
+    /**
+     * @throws EppoClientInitializationException
+     */
+    private static function createAndInitClient(
+        FlagConfigurationLoader $configLoader,
+        PollerInterface $poller,
+        ?LoggerInterface $assignmentLogger,
+        ?bool $isGracefulMode
+    ): EppoClient {
+        $instance = new self($configLoader, $poller, $assignmentLogger, $isGracefulMode);
+
+        // Load configuration on startup.
+        try {
+            $configLoader->fetchAndStoreConfigurations();
```

Comment:
**New** Start fetching on initialization.

Reply:
PHP will initialize _every_ request, though, right?

If so, we only want to do this if the polling interval has passed, otherwise use what was in the file cache.

Reply:
Rerouted config loading calls through a throttling mechanism (that uses another persistent cache)